tx-pool/src/process.rs

```diff
@@ -135,7 +155,7 @@ impl TxPoolService {
                 for id in conflicts.iter() {
                     let removed = tx_pool.pool_map.remove_entry_and_descendants(id);
                     for old in removed {
-                        debug!(
+                        eprintln!(
```

Comment:
why use eprintln here?

Reply:
it's forgotten after debug code, rollbacked.